www/main/viewer.js

```diff
@@ -1,339 +0,0 @@
-/* eslint
```

Comment:
This code was not being used, and was from the old code, so deleted entire file.